drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -820,6 +820,9 @@ void RasterizerSceneGLES3::_draw_sky(RID p_env, const Projection &p_projection,
 		float aspect = p_projection.get_aspect();
 
 		camera.set_perspective(environment_get_sky_custom_fov(p_env), aspect, near_plane, far_plane);
+		Projection correction;
+		correction.set_depth_correction(false, true, false);
+		camera = correction * camera;
```

Comment:
@dsnopek I think that this code might fix your issue. It needs to be moved outside the conditional so it applies regardless of whether sky custom fov is used. Can you test again, but with these lines moved below the conditional?

Reply:
No, unfortunately, that doesn't appear to fix it :-/